packages/perseus-core/src/parse-perseus-json/regression-tests/item-data/hint-with-null-replace.json

```diff
@@ -17,14 +17,14 @@
           "choices": [
             {
               "content": "crwdns2693864:0crwdne2693864:0",
-              "clue": "",
+              "rationale": "",
```

Comment:
This test data should not be updated.  @benchristel can explain it better, but these are v1.0 configurations and so will be migrated automatically by the parser.

What we probably want is some v3 versions of radio widgets that can form teh basis of the parser validation for v3 radio widgets.